src/test/resources/features/CompanyDetails.feature

```diff
@@ -0,0 +1,39 @@
+Feature:Company Details  - Validations
+
+  Background:
+    Given Marginal Relief Calculator is launched
+    When the user clicks on start now button
+    Then the user lands on accounting period page
+    When the accounting period start date is provided as "01/01/2023"
+    Then the user clicks continue button on accounting period page
+    And the profit is "70000"
+    Then the user clicks continue button on taxable profit page
+    And the user selects option "No" for the question Did your company receive any distributions?
+    And the user clicks continue button on distributions page
+    Then user is landed on associated companies page
+    When the user selects option "no" for the question Did your company have any active associated companies?
+    Then the user clicks continue button on associated companies page
+    Then user is presented with Check Your Answers page
+    And the user clicks on calculate marginal relief button on check you answers page
+    And the Check Marginal Relief calculation in detail link is displayed
+    And the user clicks on Print or save your calculation link
```

Comment:
What is the expected check here? It seems to stop at "the user clicks on Print or save your calculation link"

Reply:
Updated

Reply:
Aah, this is the background for other scenarios. Missed it :)